src/validator/Tests.java

```diff
@@ -0,0 +1,55 @@
+package validator;
+
+import static org.junit.Assert.*;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+
+import org.junit.Test;
+
+public class Tests {
+	private static CreditCardVendorsReader reader = new CreditCardVendorsReader();
+	private static ArrayList<CreditCardVendor> vendors = reader.returnVendorArray();
```

Comment:
It's re-using the production code to initialize the test parameter : vendor.
I would recommend to make it independent from production code
and prepare list of possible positive/negative cases.

Then test several positive or negative cases 
using @ParameterizedTest (Junit 5 annotation)
with specific method -  e.g. shouldReturnTrueForCorrectNumber